test/meters.jl

```diff
@@ -19,48 +21,73 @@ OpenDSSDirect.Text.Command("""
 @test Meters.MeteredTerminal() == 1
 @test Meters.MeteredTerminal(Meters.MeteredTerminal()) == nothing
 @test Meters.DIFilesAreOpen() == 0
-@test_broken Meters.SampleAll() == 0    # TODO: segfault
+if typeof(1) == Int64
```

Comment:
Any idea what's at the root of these 32-bit windows failures? Is it an OpenDSS_CAPI issue? Anyway, it's low priority and shouldn't block anything.

Reply:
I was getting a really nasty segfault: https://ci.appveyor.com/project/kdheepak/opendssdirect-jl/builds/22154716/job/yrg8sh7jawia20hs

I then installed x86 locally and was able to get the same segfault locally. The odd thing was I didn't get a segfault if I ran just the `test/meters.jl` file. I only got a segfault if I ran all the previous tests and then ran the `test/meters.jl` file. And even this segfault only occurs if I run things in a certain order I think. I bisected the file, commenting out blocks and narrowed it down to this function. This is definitely something that warrants further investigation.

Reply:
All those segfaults (maybe with the exception of one) are present with the official COM DLL too. 

> "Access violation at address 0000000000791F08 in module 'OpenDSSEngine.DLL'. Read of address 0000000000000008"

Weird that it only happens on Windows. I'll try to see what's happening tomorrow -- most likely all the segfaults are related to the same uninitialized variable.

Reply:
With a debug build, I was able to reproduce the issue on Linux too. The issue with the meters is that the meter zone list is not initialized. In a couple of places there was a check with the following error message:

> Meter Zone Lists need to be built. Do Solve or Makebuslist first!

I added several more checks for that, but the error number is not used in OpenDSSDirect.jl right now. At least it won't segfault...

For the tests, maybe a SolveSnap could be used in the future to emulate actual usage.